src/verinfast/user.py

```diff
@@ -25,7 +25,8 @@ def repeat_boolean_prompt(
         print_opts = "(Y/n)"
         d = "y"
 
-    logger(prompt)
+    prompt = "\n\n" + prompt
+    print(prompt)
```

Comment:
I think we wanted logger with display=True, timestamp=False so it would show in the logs.

Reply:
I tried that, but logger forced a ":" to be in the log.  I updated logger. We need (in a future release) a "logger" and "pretty printer."